doc/cascadia/profiles.schema.json

```diff
@@ -800,7 +819,7 @@
         },
         "bellStyle": {
           "default": "audible",
-          "description": "Controls what happens when the application emits a BEL character. When set to \"audible\", the Terminal will play a sound. When set to \"none\", nothing will happen.",
+          "description": "Controls what happens when the application emits a BEL character. When set to \"all\", the Terminal will play a sound and flash the taskbar icon. An array of specific behaviors can also be used. Supported array values include `audible` and `visual`. When set to \"none\", nothing will happen.",
```

Comment:
For whatever reason, my head is having a hard time parsing this description. I don't know how to do it better, but this feels weird to me. Summoning @cinnamon-msft